src/WebJobs.Script/Config/ScriptJobHostOptionsSetup.cs

```diff
@@ -51,6 +51,14 @@ public void Configure(ScriptJobHostOptions options)
                 {
                     options.FileLoggingMode = fileLoggingMode.Value;
                 }
+
+                // Default SendCanceledInvocationsToTheWorker to true if not provided
+                var sendCanceledInvocationsToTheWorker = jobHostSection.GetSection("SendCanceledInvocationsToTheWorker");
```

Comment:
Didn't see more info on how you expect this to be structured in `host.json`, but I'd expect this lookup to use `GetValue` instead of treated as a section.

Reply:
Should not be in a section:

```json
{
    "version": "2.0",
    "sendCanceledInvocationsToWorker": false,
}
```

Will switch to GetValue